packages/backend-for-frontend/.env

```diff
@@ -39,6 +40,7 @@ GENERATED_JWT_SECONDS_DURATION=60
 S3_CUSTOM_SERVER=true
 S3_SERVER_HOST=http://localhost
 S3_SERVER_PORT=9000
+S3_BUCKET=interop-local-bucket
```

Comment:
This should not be needed: We have multiple buckets in the BFF, so we created a dedicated `*_CONTAINER` variable for each bucket. This `S3_BUCKET` is still used in services that interact with a single bucket, such as some *-process services, and in my opinion in the future should be dismissed everywhere in favour of more specific config vars